client/v1/client_test.go

```diff
@@ -7,10 +7,10 @@ import (
 	"io/ioutil"
 	"net/http"
 	"net/http/httptest"
-	"reflect"
 	"testing"
 	"time"
 
+	"github.com/google/go-cmp/cmp"
```

Comment:
Why the library change?

Reply:
VS.Code linter warned about the usage of `reflect.DeepEqual`. It was probably a false warning since the `reflect.DeepEqual` caveats did not apply to the cases used in the test.  I appeased the linter, https://pkg.go.dev/github.com/google/go-cmp/cmp should be rather used in tests.